atk/src/attribute_set.rs

```diff
@@ -1,5 +1,3 @@
-// Copyright 2013-2018, The Gtk-rs Project Developers.
-// See the COPYRIGHT file at the top-level directory of this distribution.
-// Licensed under the MIT license, see the LICENSE file or <https://opensource.org/licenses/MIT>
+// Take a look at the license at the top of the repository in the LICENSE file.
```

Comment:
Also we need to fix up the COPYRIGHT and LICENSE file. Basically following https://github.com/rust-lang/rust/pull/43498